Lib/test/test_call.py

```diff
@@ -169,6 +169,36 @@ def test_oldargs1_2_kw(self):
         msg = r"count\(\) takes no keyword arguments"
         self.assertRaisesRegex(TypeError, msg, [].count, x=2, y=2)
 
+    def test_argtype0(self):
```

Comment:
A bit weird to have one test function per test but I guess that's the style of this file.

Reply:
Yes, I tried to follow the style.

Reply:
Following the style is generally nice, but not enforced; you do not have to write unnecessarily verbose code just because other tests were written like that. It would be ok to write a single compact test function with a `subTest` loop; it would mean fewer lines of code to review, and fewer lines of code to maintain. IMO that is more important than following style.